packages/web-ui/src/ds/molecules/DiffViewer/Editor/language.ts

```diff
@@ -0,0 +1,265 @@
+import { useEffect, useState } from 'react'
+
+import hslToHex from 'hsl-to-hex'
+import type { languages } from 'monaco-editor'
+import { useTheme } from 'next-themes'
+
+export const tokenizer = {
```

Comment:
damn, did you just create a whole syntax highlighting tokenizer for the diff editor theme? 👏👏

Reply:
xd
Yeah, since there is no syntax highlighting that fits Latitude I had to create my own...